### ISSUE TITLE
Anbox fails to start (Lubuntu 17.10 on virtualbox 5.2.0)

### ISSUE THREAD

**Issue by mygithubcomments2017** [anbox/anbox#543]
Make sure you are running the latest version of Anbox before reporting an issue.

Please also check that no similar bug is already reported. Have a look on the list of open bugs at https://github.com/anbox/anbox/issues

** Please paste the result of `anbox system-info` below:**
```
$ anbox system-info
libEGL warning: DRI2: failed to authenticate
...
$ 
```

**Please describe your problem:**
--> Anbox fails to start (Lubuntu 17.10 on virtualbox 5.2.0).  Workflow: Start Button -> Other -> Anbox --> Application opens, displays "Starting ..." message (with android icon) --> Crashes after 20 secs.

**What were you expecting?:**
--> Crashes after 20 secs.

**Additional info:**
`>` Lubuntu 17.10 on virtualbox 5.2 (testing out before I run an alpha version on a real hardware)
`>` More info:
```
$ ANBOX_LOG_LEVEL=debug anbox session-manager
libEGL warning: DRI2: failed to authenticate
[ 2017-12-09 16:05:39] [Renderer.cpp:165@initialize] Using a surfaceless EGL context
[ 2017-12-09 16:05:39] [Renderer.cpp:190@initialize] Failed: Missing egl_image related extension(s)
Segmentation fault (core dumped)
```

**Comment by mygithubcomments2017** [comment]
Resolved this with help of irc user 'maerco' -- via command line parameter  '--gles-driver=translator'
`>` First time running `anbox session-manager --gles-driver=translator` yielded `Segmentation fault (core dumped)`.
`>` However, on the next reboot, `anbox session-manager --gles-driver=translator` did not crash. I then waited 1-2 minutes until disk activity in vm has stopped and then ran `anbox launch --package=org.anbox.appmgr --component=org.anbox.appmgr.AppViewActivity --gles-driver=translator' in another terminal window, which popped-up android applications screen.
